docs/manual/introduction/Optimizations.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<base href="../../" />
+		<script src="list.js"></script>
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		<h1>[name]</h1>
+
+		<p class="desc">
+			This article shows how to optimize your three.js scene.
+		</p>
+
+		<h2>Metrics</h2>
+
+		<p>
+			Before starting, we need to define a some metrics, and how to monitor them. There are two important factors
+			that will affect your performances:
+
+			<ul>
+				<li>CPU: affects the number of frames per seconds (FPS) which represents how fluid is your scene</li>
+				<li>Memory: affects the memory usage which can be restricted on low devices like smartphones</li>
+			</ul>
+		</p>
+
+		<p>
+			In order to monitor them you can use [mrdoob/stats.js](https://github.com/mrdoob/stats.js) repository.
+		</p>
+
+		<h2>Buffer Geometries</h2>
+
+        <p>
+            BufferGeometries store information (such as vertex positions, face indices, normals, colors,
+            UVs, and any custom attributes) in [page:BufferAttribute buffers] - that is,
+            [link:https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays typed arrays].
+        </p>
+
+        <p>
+            If your are dealing with static scene, or a huge one and need to reduce memory usage you should rather use
+            BufferGeometries instead of Geometries which are heavier.
```

Comment:
This doesn't really depend on the static scene. `Geometry` is there mostly for convenience, it's easier to traverse it's structure than just having stuff in typed arrays.